extensions/wikia/FeedsAndPosts/FeedsAndPostsHooks.class.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+class FeedsAndPostsHooks {
+
+	/**
+	 * Checks whether the JS assets should be loaded for Feeds and Posts integration
+	 *
+	 * @return bool
+	 */
+	private static function shouldLoadAssets() {
+		return WikiaPageType::isArticlePage();
```

Comment:
This function isn't really needed but I kept it for clarity and in case we add more conditions later on.